### PR TITLE
App

### PR DESCRIPTION
**Pull request by DelRoso**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```
Comment (SpirinEgor) [inline .github/workflows/build.yml]: Run CI on all branches, no need to restrict it since it is very quick and free 🤷‍♂️

File: streamlit_app.py [hunk 2]
```diff
@@ -1,38 +1,41 @@
...
+import easyocr as ocr  # OCR
+import streamlit as st  # Web App
+from PIL import Image  # Image Processing
+import numpy as np  # Image Processing
+from easyocr import Reader
 
-"""
-# Welcome to Streamlit!
+# title
+st.title("Easy OCR - Extract Text from Images")
```
Comment (SpirinEgor) [inline streamlit_app.py]: Wrap the code with function, using global scope isn't good

**Review by SpirinEgor — CHANGES_REQUESTED** [review]
Please don't use the global scope to define your application. Use functions to declare components and build the app. This way the whole application would be easy to reuse and extend.

File: .github/workflows/build.yml (hunk 1 above)
Comment (DelRoso) [reply .github/workflows/build.yml]: Done

File: streamlit_app.py (hunk 2 above)
Comment (DelRoso) [reply streamlit_app.py]: Done

**Review by SpirinEgor — APPROVED** [review]
👍

**Pull request merged**